### ISSUE TITLE
All in same folder or a folder per release?

### ISSUE THREAD

**Issue by caterinap** [biodiversity-exploratories-synthesis/2019_grassland_functions#1]
We should decide on the organisation of the scripts, either:
1- one folder per dataset type (diversity, functions, grl, for) with all scripts to create datasets or update them.
2- one folder per "release"
I'd go for option 1 + adding a small description of each file there (e.g. "xxx file contains the code to update the dataset xxx from BexisID1 to BexisID2").
What do you prefer?

**Comment by noschenk** [comment]
I agree to go for 1+. For the diversity dataset update, I enjoyed looking at the git diff - and it seems less chaotic than many folders. And BExIS does the archiving job, too.
I also like the description - that would be in the README, right?
I think what you did with the diversity dataset "header" is very cool, too - listing all changes since the earliest version, then it is all at one place.

**Comment by caterinap** [comment]
Ok then let's try this and we can always change later. All the "intermediate" scripts correspond to internal versions of the dataset (i.e. mini releases not in Bexis). They have the same date as the datasets we use internally.
